apis/Google.Maps.AddressValidation.V1/docs/history.md

```diff
@@ -1,15 +1,5 @@
 # Version history
 
-## Version 1.0.0-beta03, released 2023-01-16
```

Comment:
And the version in apis.json and project file?

Reply:
Yikes, well done! Will do.

Reply:
Done now - PTAL.